### DISCUSSION TITLE
Support in native libraries

### DISCUSSION THREAD

**Discussion by haarts** [walt-id/.github#40] [Help]
I really like Walt.id, the ethos of open source speaks to me. But what strikes me as odd is that, when I want to create a mobile app, I am stuck with doing REST calls to a server somewhere.

What I would like is to use the Walt.id libraries directly in my Android/iOS/Windows/Mac app. Flutter/Dart would seem ideal for this, and perhaps [Kotlin Native](https://kotlinlang.org/docs/native-overview.html) would be a viable route. Are there other ideas I should try?

**Comment by waltkb** [comment]
Our long term backlog includes implementing the SSI Kit with Kotlin Multiplatform, which allows to target multiple platforms (JDK, Mobile [Android, iOS], Native, JS). However this is a future task for now.
